app/views/cases/offender_sar/_case_details.html.slim

```diff
@@ -7,30 +7,32 @@
 
     table
       tbody.sar-basic-details
+
         = render partial: 'cases/shared/case_type', locals: { case_details: case_details }
-        = render partial: 'cases/offender_sar/data_subject', locals: { case_details: case_details }
-        = render partial: 'cases/offender_sar/subject_aliases', locals: { case_details: case_details }
-        = render partial: 'cases/offender_sar/date_of_birth', locals: { case_details: case_details }
-        = render partial: 'cases/offender_sar/prison_number', locals: { case_details: case_details }
-        = render partial: 'cases/offender_sar/previous_case_numbers', locals: { case_details: case_details }
-        = render partial: 'cases/offender_sar/other_subject_ids', locals: { case_details: case_details }
-        = render partial: 'cases/offender_sar/case_reference_number', locals: { case_details: case_details }
-        = render partial: 'cases/sar/subject_type', locals: { case_details: case_details }
-        = render partial: 'cases/offender_sar/subject_address', locals: { case_details: case_details }
-        = render partial: 'cases/offender_sar/high_profile_subject', locals: { case_details: case_details }
-        = render partial: 'cases/offender_sar/third_party', locals: { case_details: case_details }
-        - if case_details.third_party?
-          = render partial: 'cases/sar/third_party_relationship', locals: { case_details: case_details }
-          = render partial: 'cases/offender_sar/requester_name', locals: { case_details: case_details }
-          = render partial: 'cases/offender_sar/third_party_company_name', locals: { case_details: case_details }
-        = render partial: 'cases/offender_sar/response_address', locals: { case_details: case_details }
-        = render partial: 'cases/offender_sar/request_dated', locals: { case_details: case_details }
-        = render partial: 'cases/offender_sar/requester_reference', locals: { case_details: case_details }
+
+        = render partial: 'cases/offender_sar/subject_details', locals: { case_details: case_details }
+
+        = render partial: 'cases/offender_sar/requested_on_behalf', locals: { case_details: case_details }
+
+        - if case_details.third_party? && (case_details.recipient == 'requester_recipient' || case_details.recipient == 'subject_recipient')
+          = render partial: 'cases/offender_sar/third_party_details', locals: { case_details: case_details }
+
+        = render partial: 'cases/offender_sar/request_recipient', locals: { case_details: case_details }
+
+        - if (not case_details.third_party) && case_details.recipient == 'third_party_recipient'
```

Comment:
We don't need to check for third party requester here, I think? Also, the enums for the recipient generate free predicate methods, so we can write the above as simply 
```suggestion
        - if case_details.third_party_recipient?
```

Reply:
Done!